glib/src/wrapper.rs

```diff
@@ -329,144 +329,156 @@ macro_rules! glib_wrapper {
     // Object, no class struct, no parents or interfaces
     (
         $(#[$attr:meta])*
-        pub struct $name:ident(Object<$ffi_name:ty, $rust_class_name:ident>);
+        pub struct $name:ident(Object<$ffi_name:ty $(, $rust_class_name:ident)?>);
 
         match fn {
             get_type => || $get_type_expr:expr,
         }
     ) => {
-        $crate::glib_object_wrapper!(@object [$($attr)*] $name, $ffi_name, ::std::os::raw::c_void, $rust_class_name, @get_type $get_type_expr, @extends [], @implements []);
+        $crate::glib_object_wrapper!(@object [$($attr)*] $name, $ffi_name, ::std::os::raw::c_void, @get_type $get_type_expr, @extends [], @implements []);
```

Comment:
@ids1024 Can you also do the `gir` change that makes it use the new variants without the explicit class name?

Reply:
It doesn't look complicated to update `gir`, but I'm seeing issues running `generator.py` with `gir` master similar to those mentioned in https://github.com/gtk-rs/gtk-rs/pull/11

Reply:
Indeed, something's broken currently. Can you create an issue for gir otherwise?